plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -119,22 +106,151 @@ const GetStartedButton = ( { setIsError }: { setIsError: Function } ) => {
 	);
 };
 
-const SubscriptionsPage = () => {
-	const [ isError, setIsError ] = useState( false );
+const StepNumber: React.FC = ( { children } ) => (
+	<span className="wcpay-empty-subscriptions-page-step-number">
+		{ children }
+	</span>
+);
+
+const TermsOfService = () => (
+	<span className="wcpay-empty-subscriptions-page-terms-of-service">
+		{ createInterpolateElement(
+			__(
+				'By clicking “Get started”, the WooCommerce Payments plugin will be installed and you agree to the <a>Terms of Service</a>',
+				'woocommerce'
+			),
+			{
+				a: (
+					// eslint-disable-next-line jsx-a11y/anchor-has-content
+					<a
+						href="https://wordpress.com/tos/"
+						target="_blank"
+						rel="noreferrer"
+					/>
+				),
+			}
+		) }
+	</span>
+);
 
+type MainContentProps = {
+	setHasError: setHasErrorFunction;
+};
+
+const MainContent: React.FC< MainContentProps > = ( { setHasError } ) => {
 	return (
-		<div className="wcpay-empty-subscriptions__container">
-			<ErrorNotice isError={ isError } />
-			<img src={ unconnectedImage } alt="" />
-			<p className="wcpay-empty-subscriptions__description">
+		<>
+			<h2>
+				{ __( 'Start selling subscriptions today', 'woocommerce' ) }
+			</h2>
+			<p>
 				{ __(
-					'Track recurring revenue and manage active subscriptions directly from your store’s dashboard — powered by WooCommerce Payments.',
+					'With WooCommerce Payments, you can sell subscriptions with no setup costs or monthly fees. Create subscription products, track recurring revenue, and manages subscriptions directly from your store’s dashboard.',
```

Comment:
Small update to this text to match the [design](https://user-images.githubusercontent.com/1527164/166624156-5449bdca-7423-4f1b-a771-fc10bf086180.png) on #32691 

```suggestion
					'With WooCommerce Payments, you can sell subscriptions with no setup costs or monthly fees. Create subscription products, track recurring revenue, and manage subscribers directly from your store’s dashboard.',
```

Reply:
Nice catch! Fixed in b223ed2